cylc/flow/task_pool.py

```diff
@@ -1329,7 +1329,9 @@ def check_abort_on_task_fails(self):
         """
         return self.abort_task_failed
 
-    def spawn_on_output(self, itask, output, forced=False):
+    def spawn_on_output(
+        self, itask: TaskProxy, output: str, forced: bool = False
```

Comment:
Actually, I have discovered that `forced` is not used at all 🤔

It only has coverage because there is an integration test that covers it

https://github.com/cylc/cylc-flow/blob/1c5db24c7f98cf9ce749959200732e93431f89d8/tests/integration/test_task_pool.py#L1071

but it is not actually used.

Reply:
From a quick look, this method used to be called from the `cylc set` logic but no longer is. So you might as well remove the `forced` flag in this PR.